tabular/src/autogluon/tabular/models/lgb/lgb_model.py

```diff
@@ -87,9 +87,7 @@ def _fit(self, X_train=None, y_train=None, X_val=None, y_val=None, dataset_train
                 # TODO: lightgbm must have a special install to support GPU: https://github.com/Microsoft/LightGBM/tree/master/python-package#build-gpu-version
                 #  Before enabling GPU, we should add code to detect that GPU-enabled version is installed and that a valid GPU exists.
                 #  GPU training heavily alters accuracy, often in a negative manner. We will have to be careful about when to use GPU.
```

Comment:
you may want to report this to users as warning as well, that they may get better accuracy with CPU (I don't think people would expect that).

Reply:
added logging statements for this in LightGBM and CatBoost